### ISSUE TITLE
Have 500 error on root page when have "root" route

### ISSUE THREAD

**Issue by mluts** [Sontsevolosa/lunchez#16]
```
Internal Server Error

Missing `secret_token` and `secret_key_base` for 'development' environment, set these values in `config/secrets.yml`
WEBrick/1.3.1 (Ruby/2.3.1/2016-04-26) at ab:3000
```

Please add config/secrets.yml.example for providing example of secrets.yml.

Commit default `secret_token` and `secret_key_base` values for `development` and `test` environment to `config/secrets.yml.example`. Take these values (`secret_token`, `secret_key_base`) from environment variables for `production` environment in `config/secrets.yml.example`

**Comment by Sontsevolosa** [comment]
@mluts where did you find this bug?
what priority does this bug have in my work with this project ?

**Comment by Sontsevolosa** [comment]
@ I don't understand your comment

**Comment by Sontsevolosa** [comment]
Issue is resolved in #18 pull request

**Comment by Sontsevolosa** [comment]
for checkout of error:

def index
  render inline: '<p>hello world</p>'
end
